nlp/build-model-worker/README.md

```diff
@@ -0,0 +1,81 @@
+# Tock - Worker
+
+This module provides classes used to launch a tock worker
+
+Tock worker supports several build modes :
+
+ - COMMAND_LINE
+ - VERTICLE
+ - ON_DEMAND
+
+Select a mode with this property (default "VERTICLE"):
```

Comment:
We have in the code `BuildMode.valueOf(property("tock_build_worker_mode", "COMMAND_LINE"))`